geode-core/build.gradle

```diff
@@ -204,6 +204,8 @@ task createVersionPropertiesFile {
   outputs.file propertiesFile
   inputs.dir "../.git/index"
 
+  mustRunAfter(compileJava)
```

Comment:
This should not be necessary anymore since it only depends on git index.

Reply:
Fixed, committed, and rebased to `origin/develop`